atomix/cluster/src/main/java/io/atomix/raft/cluster/impl/RaftMemberContext.java

```diff
@@ -96,8 +100,9 @@ public void resetState(final RaftLog log) {
   public boolean canAppend() {
     return appending == 0
         || (appendSucceeded
-            && appending < MAX_APPENDS
-            && System.currentTimeMillis() - (timeStats.getMean() / MAX_APPENDS) >= appendTime);
+            && appending < maxAppendsPerMember
```

Comment:
Nit: maybe we can call `appending` instead `inFlightAppendCount`? Just from the name, I kind of expected it to be a boolean flag :sweat_smile: I know it was already like this, but if we're already changing this code... :eyes:

Reply:
:laughing: yeah I can change that as well. I just replaced the occurrences tbh